clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -70,6 +70,11 @@ export default async function handler(
     }));
   }
 
+  // DEFER: The Fides server controls whether or not TCF is enabled, and so we
+  // should prefetch that value here. Until the endpoint is ready, we use an
+  // environment variable
+  const tcfEnabled = environment.settings.TCF_ENABLED;
```

Comment:
This is a little weird and we might want to think about this more—but whether TCF is enabled will be determined by the backend, not via an an option or an environment variable. Of course, we can have an option to override it, but I was thinking the PC would need to prefetch if we are using TCF from the Fides server, then determine which bundle to use. Since that determines which bundle to use, the bundles themselves don't respect the `tcfEnabled` option since they can't just switch themselves out with another bundle...